curated_transformers/tokenization/hf_loader.py

```diff
@@ -26,7 +26,9 @@ def _convert_encoder(model: Model, tokenizer: "transformers.PreTrainedTokenizerB
         return _convert_wordpiece_encoder(model, tokenizer)
     elif isinstance(tokenizer, transformers.RobertaTokenizerFast):
         return _convert_byte_bpe_encoder(model, tokenizer)
-    elif isinstance(tokenizer, transformers.XLMRobertaTokenizerFast):
+    elif isinstance(tokenizer, transformers.XLMRobertaTokenizerFast) or isinstance(
+        tokenizer, transformers.CamembertTokenizerFast
+    ):
```

Comment:
```suggestion
    elif isinstance(tokenizer, (transformers.XLMRobertaTokenizerFast, transformers.CamembertTokenizerFast)):
```

Reply:
Oh, I didn't know that you can provide a tuple. That's really cool. 🎉